### PR TITLE
feat: add unfreeze collective action button to banner

### PR DESCRIPTION
**Pull request by snitin315**
Resolve  https://github.com/opencollective/opencollective/issues/5540
 

# Description

# Screenshots

Screenshot 2022-08-19 at 2 52 09 PM

https://user-images.githubusercontent.com/46647141/185588084-e0152888-5f04-49b8-ada0-09f1b12e0b7f.mov

### CODE REVIEW

**Review by Betree — COMMENTED** [review]
Looks great so far! The banner doesn't disappear after submitting the modal, can we do something about it? Maybe [update cache](https://www.apollographql.com/docs/react/caching/cache-interaction/) or refetch query?

**Comment by snitin315** [comment]
@Betree Done, I used `refetchQueries`.

https://user-images.githubusercontent.com/46647141/186675902-0ad703fc-3124-4077-a10b-941e471da7f4.mov

File: components/GlobalWarnings.js [hunk 1]
```diff
@@ -25,8 +27,13 @@ const GlobalWarningContainer = styled.div`
  */
 const GlobalWarnings = ({ collective }) => {
   const { LoggedInUser } = useLoggedInUser();
+  const [hasFreezeModal, setHasFreezeModal] = React.useState(false);
 
   if (collective?.isFrozen) {
+    const isLoggedInUserHostAdmin = LoggedInUser?.memberOf?.some(
+      member => collective.host?.id === member?.collective?.id,
+    );
```
Comment (Betree) [inline components/GlobalWarnings.js:35]: If `collective.host` is null, then we can end up with `null === null` which is true. Let's make it simpler & safer:

```suggestion
    const isLoggedInUserHostAdmin = Boolean(LoggedInUser?.isHostAdmin(collective));
```

**Review by Betree — APPROVED** [review]
Works great, thanks!

**Pull request merged**